MAINTAINERS.yml

```diff
@@ -5661,6 +5661,21 @@ Peregrine Platforms:
   labels:
     - "platform: Peregrine"
 
+WCH Platforms:
+  status: maintained
+  maintainers:
+    - nzmichaelh
+    - kholia
+    - VynDragon
```

Comment:
Can this be split into maintainers/collaborators?

Reply:
Done.